apps/epp-amend/sections/summary-data-sections.js

```diff
@@ -9,5 +9,17 @@ module.exports = {
         field: 'amend-licence-number'
       }
     ]
+  },
+  'amend-contact-details': {
+    steps: [
+      {
+        steps: '/amend-contact-details',
```

Comment:
if path name has changed even step name should change. also in other forms I noticed that the when a step contains multiple fields the structure looks something like this :
```
steps:[
        {
            step: ' ....',
            field: ' ...'
        },
        {
            step: ' ....',
            field: ' ...'
        }
]